app/src/main/java/org/oppia/android/app/devoptions/vieweventlogs/EventLogItemViewModel.kt

```diff
@@ -24,10 +24,7 @@ class EventLogItemViewModel(
 
   /** Returns the event log context in a human readable format. */
   fun formatContextString(): String =
-    eventLog.context.activityContextCase.name.capitalizeWords().substringBeforeLast(" ")
-
-  /** Returns the event log action name in a human readable format. */
-  fun formatActionNameString(): String = eventLog.actionName.name.capitalizeWords()
+    eventLog.context.activityContextCase.name.capitalizeWords()
```

Comment:
Will this fit on one line with the fun declaration?

Also, does this have the correct behavior? I'd expect the enum names to be in UPPER_SNAKE_CASE so presumably capitalizeWords() won't have any effect.

Reply:
The expected behaviour is `Upper Snake Case` from `UPPER_SNAKE_CASE`, right? If yes, then capitalizeWords() does work.

Reply:
Ah interesting. I didn't know it extract words from snake case.